### ISSUE TITLE
gpu sharing on summit

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/radical.pilot#1789]
Summit may be able to do GPU sharing.  AM to inquire for documentation.  How does that map to out `gpu_threads`?

**Comment by mturilli** [comment]
Ongoing.

**Comment by mturilli** [comment]
Stumbled upon an underlining problem. Open a ticket with ORNL [CCS #398578]

**Comment by mturilli** [comment]
depends on #1818

**Comment by mturilli** [comment]
To be addressed by using GPU oversubscription.

**Comment by mturilli** [comment]
Addressed by the ERF syntax. Requires API change, parked until real use case emerges.
